### ISSUE TITLE
Converting input to an suggest object on button press

### ISSUE THREAD

**Issue by mskwarcan** [ubilabs/react-geosuggest#284]
I have an app that allows people to begin typing in an address to start looking for locations. Currently if you start typing and click one of the suggestions or press enter, it performs a search and 
the onSuggestSelect function fires off which returns and object that has the location parameter which contains the lat/lng. This works great.

The issue I'm running into is that I also have a search button that the user can click on. If the user begins typing and doesn't hit enter or choose a suggestion, than the address isn't parsed and an object with location information isn't returned. I'm not sure why it would work by submitting the form with enter vs pressing a submit button.

I tried getting around by setting the autoActivateFirstSuggest to true and grabbing the first suggestion based on the input into the textbox, but unforunately all this returns is a label and not the location object (lat/lng).

In short:

1. Type in a zip code
2. Don't choose a selection or hit enter
3. Use a submit button to submit the form
4. An object isn't returned, but the plain text in the box.

The goal is to allow users to quickly type in their zip codes and hit the submit button, but that doesn't work at the moment.

**Comment by ro-ka** [comment]
What could a solution to this be? We can’t hook into the submit of a parent form…

**Comment by ro-ka** [comment]
Closing this as there is no feedback. Reopen if needed.
